config/initializers/devise_async.rb

```diff
@@ -0,0 +1,3 @@
+Devise::Async.setup do |config|
```

Comment:
can we use only `devise.rb` file for this?

Reply:
Yes, I corrected this